gcp/modules/gke-cluster/main.tf

```diff
@@ -19,7 +19,7 @@ module "gke_cluster" {
 
   initial_node_count = 1
   node_type          = "${var.node_type}"
-  kubernetes_version = "1.10.7-gke.6"
+  kubernetes_version = "1.10.9-gke.3"
```

Comment:
I am eager to try 1.11, and looks like `1.11.2-gke.15` is now available. Do you think we should still stick to 1.10?

Reply:
Let's do 1.10 for now as it should be straightforward and I'm happy to try 1.11, but let's do it once our pipeline is green